src/main/java/com/addthis/meshy/MeshyServer.java

```diff
@@ -384,6 +388,14 @@ public void connectPeer(InetSocketAddress address) {
         }
     }
 
+    public void connectPeers(List<InetSocketAddress> addresses) {
+        connectPeers(addresses, retryPeersUnitDelay, retryPeersMaxDelayUnit);
+    }
+
+    public void connectPeers(List<InetSocketAddress> addresses, int unitDelay, int maxDelayUnit) {
```

Comment:
I think we should move the startAutoConnectToPeers() call into the MeshyServer constructor:
`if (automesh) {startAutoMesh} else {startAutoConnectToPeers}`

I think if we do this, we can undo all the changes to meshy/hydra/mqmaster (except for upgrading meshy version) and get rid of both connectPeers methods.

What do you think?